scripts/final_glue_runs.sh

```diff
@@ -132,14 +132,6 @@ JIANT_OVERRIDES="pretrain_tasks = dissentwikifullbig, run_name = dissent-elmo, e
 
 ## MSCOCO ##
 
-# Sam is running.
```

Comment:
Probably better to leave this file as a record, though we will have to move it elsewhere eventually. How about commenting these out and marking them as deprecated?

Reply:
Restored + commented out with deprecation comment.